### ISSUE TITLE
Bevel border

### ISSUE THREAD

**Issue by jaumard** [Idean/Flutter-Neumorphic#119]
Material have something like this:
```
shape: const BeveledRectangleBorder(
                  borderRadius: BorderRadiusDirectional.only(topStart: Radius.circular(38), topEnd: Radius.circular(38)),
                ),
```
Would be nice to have the equivalent here, something like: 
```
boxShape: NeumorphicBoxShape.bevelRect(borderRadius: BorderRadius.circular(12)),   
or
boxShape: NeumorphicBoxShape.bevelRect(borderRadius: BorderRadiusDirectional.only(topStart: Radius.circular(38), topEnd: Radius.circular(38))),
```

Workaround for now:
use custom path as it's already supported, but more complicated to setup.

**Comment by florent37** [comment]
yes we can add a bevel border directly inside the neumorphic library
we will implement this in next release

**Comment by florent37** [comment]
I added this 

```
boxShape = NeumorphicBoxShape.beveled(
                        BorderRadius.circular(12)
);
```

**Comment by florent37** [comment]
published in 2.2.1
